app/javascript/stylesheets/components/button.scss

```diff
@@ -0,0 +1,66 @@
+.btn {
+  padding: 0.5rem 1.25rem;
+  border-radius: 5px;
+  outline: none;
```

Comment:
**issue (blocking)**: when you remove the outline of an element you should define the focus behaviour with `:focus` otherwise someone navigating the website with keyboard only (using tab) it's not possible to know the element that is currently in focus.
Also add this in `.login-input`